css/selectors4/focus-within-006.html

```diff
@@ -5,14 +5,10 @@
 <link rel="author" title="Keyong Li" href="mailto:kli79@bloomberg.net">
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
-<link rel="match" href="focus-within-001-ref.html">
+<link rel="match" href="focus-within-006-ref.html">
 <meta name="assert" content="Test that :focus-within works on form controls, using an input element.">
 <style>
-/* Suppress things that cannot be reproduced in the reference file */
 :focus {
-  all: initial;
-  outline: none;
```

Comment:
`outline: none;` needs to stay (and to be in the reference as well), otherwise “There must be no red or blue once it is focused.” isn't true, since the outline is blue, at least on some platforms.